emsdk_manifest.json

```diff
@@ -260,43 +260,43 @@
   },
   {
     "id": "node",
-    "version": "14.15.5",
+    "version": "14.18.2",
     "bitness": 32,
     "arch": "x86",
-    "windows_url": "node-v14.15.5-win-x86.zip",
+    "windows_url": "node-v14.18.2-win-x86.zip",
     "activated_path": "%installation_dir%/bin",
     "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
   },
   {
     "id": "node",
-    "version": "14.15.5",
+    "version": "14.18.2",
     "arch": "arm",
     "bitness": 32,
-    "linux_url": "node-v14.15.5-linux-armv7l.tar.xz",
+    "linux_url": "node-v14.18.2-linux-armv7l.tar.xz",
     "activated_path": "%installation_dir%/bin",
     "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
   },
   {
     "id": "node",
-    "version": "14.15.5",
+    "version": "14.18.2",
     "bitness": 64,
     "arch": "x86_64",
-    "macos_url": "node-v14.15.5-darwin-x64.tar.gz",
-    "windows_url": "node-v14.15.5-win-x64.zip",
-    "linux_url": "node-v14.15.5-linux-x64.tar.xz",
+    "macos_url": "node-v14.18.2-darwin-x64.tar.gz",
+    "windows_url": "node-v14.18.2-win-x64.zip",
+    "linux_url": "node-v14.18.2-linux-x64.tar.xz",
     "activated_path": "%installation_dir%/bin",
     "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
   },
   {
     "id": "node",
-    "version": "14.15.5",
+    "version": "14.18.2",
     "arch": "aarch64",
     "bitness": 64,
-    "macos_url": "node-v14.15.5-darwin-x64.tar.gz",
-    "linux_url": "node-v14.15.5-linux-arm64.tar.xz",
+    "macos_url": "node-v14.18.2-darwin-x64.tar.gz",
+    "linux_url": "node-v14.18.2-linux-arm64.tar.xz",
     "activated_path": "%installation_dir%/bin",
     "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
     "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
```

Comment:
We should not change the existing declarations from 14.15.5 to 14.18.2, but instead add new declarations. Otherwise if people have installer batch scripts that do `emsdk install node-14.15.6-64bit`, they will break.

Reply:
Done